test/unit/ready.js

```diff
@@ -67,4 +70,20 @@ QUnit.module( "ready" );
 			"Argument passed to fn in jQuery(document).ready( fn ) should be jQuery" );
 	} );
 
+	QUnit.test( "Promise.resolve(jQuery.ready)", function( assert ) {
+		assert.expect( 2 );
+
+		var beforeReady = assert.async();
+		var afterReady = assert.async();
```

Comment:
Hmm. @gibson042, to what would we have to bump?

It's probably better to fix all those cases at the same time. But, @timmywil, to be consistent with the rest of the tests and to make it easier to find all such code in one find+replace you could use the pattern from:
https://github.com/jquery/jquery/blob/250fd800a171cac9ab713cbbe1a70a90a348e3dc/test/unit/deferred.js#L463
and:
https://github.com/jquery/jquery/blob/250fd800a171cac9ab713cbbe1a70a90a348e3dc/test/unit/deferred.js#L470
i.e.:
```js
var done = jQuery.map( new Array( 2 ), function() { return assert.async(); } );
done.pop().call();
```